samples/openmp_testsuite/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG IMAGE_FLAVOR=alpine
+ARG IMAGE_VERSION=3.8
+
+FROM ${IMAGE_FLAVOR}:${IMAGE_VERSION} AS builder
+RUN wget -q https://ftp.gnu.org/gnu/gcc/gcc-7.5.0/gcc-7.5.0.tar.gz && \
+    tar -xzf gcc-7.5.0.tar.gz && \
+    rm -f gcc-7.5.0.tar.gz
+RUN apk add --quiet --no-cache bash build-base dejagnu isl-dev make mpc1-dev mpfr-dev texinfo zlib-dev
+WORKDIR /gcc-7.5.0/libgomp/testsuite
+COPY tests.mak .
+RUN make -f tests.mak
+
+FROM ${IMAGE_FLAVOR}:${IMAGE_VERSION}
+RUN apk add --quiet --no-cache bash build-base
```

Comment:
Is there a reason why build-base is still needed in the final image?

Reply:
![image](https://user-images.githubusercontent.com/9376183/123467971-00d24600-d5a6-11eb-8360-1cc8ef9a9767.png)
Some of the test cases will fail with this error.